net/arp-scan/Makefile

```diff
@@ -41,6 +41,10 @@ endef
 define Package/arp-scan/install
 	$(INSTALL_DIR) $(1)/usr/bin
 	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/arp-scan $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/usr/share/arp-scan
+	$(INSTALL_DATA) $(PKG_BUILD_DIR)/ieee-iab.txt $(1)/usr/share/arp-scan/
+	$(INSTALL_DATA) $(PKG_BUILD_DIR)/ieee-oui.txt $(1)/usr/share/arp-scan/
+	$(INSTALL_DATA) $(PKG_BUILD_DIR)/mac-vendor.txt $(1)/usr/share/arp-scan/
```

Comment:
Make´s this sense to add this additional files?
They are quit big for a small device.
ieee-iab.txt 137kB 
ieee-oui.txt 783kB
I think we should move this files to a separate package.

Reply:
@feckert the essential functionality of `arp-scan` tool is in immediate indication of a vendor/manufacturer for each discovered MAC; without this information an output is almost useless. I agree that the database data is quite large for small devices. I see two possible approaches: 1) make separate package with the database, 2) add a couple of scripts - https://github.com/royhills/arp-scan/blob/master/get-iab and https://github.com/royhills/arp-scan/blob/master/get-oui to the base package and provide a note (does `opkg` have something like `post-install` scripts?..) containing the instructions on how to fetch fresh MAC database data to the device using those scripts.

What do you think?

Another possible extension would be to allow `arp-scan` to use gzipped data, but this definitely is separate topic. :)

Reply:
Uhm, those updater scripts are written in perl, and I suppose there's no any sense in adding a perl dependency to the arp-scan package... Perhaps separate package with MAC database is the best option as of now. @urusha any thoughts?